config/peerpods/podvm/bootc/Containerfile.rhel

```diff
@@ -1,5 +1,5 @@
 # build pause
-FROM registry.k8s.io/pause:3.9 as pause-bin
+FROM registry.k8s.io/pause:3.10 as pause-bin
```

Comment:
@snir911 - an update suggested by konflux. I have no idea whether it's good or bad. What do you think?